OrbitCore/BlockChain.h

```diff
@@ -40,9 +39,9 @@ struct Block {
 
   Block<T, Size>* m_Prev;
   Block<T, Size>* m_Next;
-  T m_Data[Size];
   BlockChain<T, Size>* m_Chain;
-  std::atomic<uint32_t> m_Size;
+  alignas(4) uint32_t m_Size;
```

Comment:
why do you need alignas here and below? isn't it aligned by default?

Reply:
I wasn't sure it would be guaranteed.  Perhaps it is, but it doesn't hurt to be explicit.

Reply:
it is guaranteed as long as structure is not packed and it is not.

Reply:
Right, ok, I'll remove as it's redundant then.